src/common-lib/schemas/onboarding.ts

```diff
@@ -0,0 +1,6 @@
+import { z } from "zod";
+
+export const onboardingSchema = z.object({
+  "owa:isTeacher": z.boolean(),
```

Comment:
I don't think we need to send details of the user's school to Clerk. I'm thinking Clerk only needs to know whether the user is a teacher or not and the other details belong in Hubspot.

Reply:
shall we add source app into clerk as well?

Reply:
yeah, I think we'll probably have some posthog stuff to do to capture sign up events and what not right? but it feels like a good thing to capture so that we can do some rudimentary attribution of sign-ups. I'll add it to this PR as it's a minor change.